src/variables/index.ts

```diff
@@ -96,17 +112,29 @@ export namespace Variables {
     set variables(variables: IVariable[]) {
       this._currentScope.variables = variables;
       this._changed.emit();
+
+    }
+
+    get variableExpanded(): ISignal<this, IVariable> {
+      return this._variableExpanded;
     }
 
     getCurrentVariables(): IVariable[] {
       return this.variables;
     }
 
+    getMoreDataOfVariable(variable: IVariable) {
```

Comment:
Maybe this could be called `expandVariable` if what it does is just emitting `_variableExpanded`?

Reply:
Or can be onExpandVariable?